rllib/models/catalog.py

```diff
@@ -63,6 +62,26 @@
     # Experimental (only works with `_use_trajectory_view_api`=True):
     # Whether the LSTM is time-major (TxBx..) or batch-major (BxTx..).
     "_time_major": False,
+    
+    # == Attention Nets (experimental: torch-version is untested) ==
+    # Whether to use a GTrXL ("Gru transformer XL"; attention net) as the
+    # wrapper Model around the default Model.
+    "use_attention": False,
```

Comment:
Why not move these configs to custom model?

Reply:
I would like for this to work exactly the same as LSTMs (attention should become a drop-in replacement for LSTMs). So it must be completely automatic to wrap any custom(!) model with an attention "head". Otherwise, you would have to mix (manually) the attention net (which is a model in itself, other than LSTM, which is a keras layer) with your custom model.